src/Runner.Worker/ActionRunner.cs

```diff
@@ -296,7 +296,7 @@ private string GenerateDisplayName(ActionStep action, DictionaryContextData cont
             var templateEvaluator = context.ToPipelineTemplateEvaluator();
             try
             {
-                didFullyEvaluate = templateEvaluator.TryEvaluateStepDisplayName(tokenToParse, contextData, out displayName);
+                didFullyEvaluate = templateEvaluator.TryEvaluateStepDisplayName(tokenToParse, contextData, context.ExpressionFunctions, out displayName);
```

Comment:
look into this more... one of the goals of this pr is to fix hash files display name bug